jsm-aug2023.qmd

```diff
@@ -0,0 +1,237 @@
+:::
+:::
+
+## Unit Testing
+
+- Unit tests in `mmrm` are transparent, compared to `PROC MIXED`.
+- Uses the `testthat` framework with `covr` to communicate the testing coverage.
+- Unit tests can be found in the GitHub repository under [./tests](https://github.com/openpharma/mmrm/tree/main/tests/testthat).
+
+::: callout-note
+The integration tests in `mmrm` are set to a tolerance of 10e\^-3 when compared to SAS outputs.
+:::
+
+## Estimation Methods
+
+| Method | `{mmrm}` | `PROC MIXED` |
+|:-------|:--------:|:------------:|
+| ML     |    X     |      X       |
+| REML   |    X     |      X       |
+
+## Covariance structures
+
+::: columns
+::: {.column width="60%"}
+- SAS has 23 non-spatial covariance structures, while mmrm has 10.
+  - 9 structures intersect with SAS
+  - Ante-dependence (homogeneous) is only in the mmrm package.
+- SAS has 14 spatial covariance structures compared to the spatial exponential one available in mmrm.
+:::
+
+::: {.column width="40%"}
+::: callout-tip
+For users that need more structures, `mmrm` is easily extensible - let us know via [feature requests](https://github.com/openpharma/mmrm/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) in the GitHub repository.
+:::
+:::
+:::
+
+## Covariance structures details
+
+|       Covariance structures                   | `{mmrm}`   |  `PROC MIXED`  |
+|:----------------------------------------------|:------------:|:-----:|
+|     Unstructured (Unweighted/Weighted)        |     X/X      |  X/X  |
+|     Toeplitz (hetero/homo)      |     X/X      |  X/X  |
+| Compound symmetry (hetero/homo) |     X/X      |  X/X  |
+|  Auto-regressive (hetero/homo)  |     X/X      |  X/X  |
+|  Ante-dependence (hetero/homo)  |     X/X      |  X    |
+|        Spatial exponential      |     X        |  X    |
+
+## Degrees of Freedom Methods
+
+|       Method     |  `{mmrm}`      | `PROC MIXED` |
+|:---------------- |:----------:|:---:|
+|       Contain         |     X*     |  X  |
+|       Between/Within  |     X*     |  X  |
+|       Residual        |     X*     |  X  |
+|       Satterthwaite   |     X      |  X  |
+|       Kenward-Roger   |     X      |  X  |
+|       Kenward-Roger (Linear)**   |  X|  X  |
+
+*Available through the `emmeans` package.
+
+**This is not identical to the KR(Lin) in `PROC MIXED`
+
+## Contrasts/LSMEANS
+
+Contrasts and least square means estimates are available in `mmrm` using:
+
+  - `mmrm::df_1d`, `mmrm::df_md` 
+  - S3 method that is compatible with `emmeans`
+  - LS means difference can be produced through `emmeans` (`pairs` method)
+  - Degrees of freedom method is passed from `mmrm` to `emmeans`
+  - By default `PROC MIXED` and `mmrm` do not adjust for multiplicity, whereas `emmeans` does.
+
+## Benchmarking with other R packages
+
+- Compared `mmrm::mmrm` with `nlme::gls`, `lme4::lmer`, `glmmTMB::glmmTMB`
+- Highlights 
+  - `mmrm` has faster convergence time

```

Comment:
maybe give indication of the difference? i.e. summarizing a bit more details of https://openpharma.github.io/mmrm/main/articles/mmrm_review_methods.html#convergence-times